nexus-repository-r-it/src/test/java/org/sonatype/nexus/plugins/r/internal/cleanup/CleanupTaskRIT.java

```diff
@@ -118,6 +120,9 @@ private int deployArtifacts(final String... names) {
             is(OK));
       }
 
+      // Wait till metadata is recreated
+      Thread.sleep(1000);
```

Comment:
Same as the above

Reply:
done in fe40919080a00ea9162d0af9be02c414af338e27